src/ui/templates/search/search.hbs

```diff
@@ -100,6 +102,10 @@
         </button>
       </div>
     {{/if}}
-    <div id="{{autocompleteContainerIdName}}" class="yxt-SearchBar-autocomplete"></div>
+    <div
+      id="{{autocompleteContainerIdName}}"
+      class="yxt-SearchBar-autocomplete"
+      role="listbox">
```

Comment:
Will having a role here conflict with this? https://github.com/yext/answers-search-ui/blob/782a818271c800792c56901136b28d59992080fe/src/ui/templates/search/autocomplete.hbs#L42
We have a `role="listbox"` already within the Autocomplete Component itself.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role

Reply:
Do you know how I might refactor to all the id to be injected to the autocomplete?

Reply:
actually it looks like I can hoist `listbox` and change the role on the autocomplete `ul` to `group`, as the current code appears to support grouping of options